client/src/game/types/NotesState.ts

```diff
@@ -0,0 +1,7 @@
+import CardNote from "./CardNote";
+import SpectatorNote from "./SpectatorNote";
+
+export default interface NotesState {
+  readonly ourNotes: readonly CardNote[];
```

Comment:
Is this indexed by order? If so maybe add a comment so that people don't think it's a simple list.

Reply:
should be a Map<number, poop> if it is indexed by order

Reply:
> should be a Map<number, poop> if it is indexed by order

Why? That would be less performant

Reply:
actually nevermind we can only have simple data structures in the state, i forgot